cluster/examples/kubernetes/ceph/object-bucket-claim-notification.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-notification-bucket
+  labels:
+    unrelated: value
+    bucket-notification: ignored
```

Comment:
Is the valid label to ignore/remove all the existing notifications for the bucket?
I

Reply:
if there are no valid labels then all notifications are removed

Reply:
I can see how this might be confusing. Is this merely here to explain that a label called `bucket-notification` is ignored by Rook for matching CephOBNs? If so, perhaps a comment would be better suited for that purpose so this doesn't confuse others as well.